site.mk

```diff
@@ -15,19 +14,11 @@ GLUON_FEATURES := \
 	web-private-wifi \
 	web-logging \
 
-
 GLUON_SITE_PACKAGES := \
 	iwinfo \
-	iptables \
 	haveged \
-	ffffm-banner_legacy \
-	ffffm-keep-radio-channel \
```

Comment:
This has the potential of breaking mesh-links on update. You probably need to write a migration package to remove persisting of channels on e.g. a pending scheduled domain-switch.